src/components/Admin/JsonEditor.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react'
```

Comment:
Is this json editor different to the Json editor in the other PR?

https://github.com/openmsupply/conforma-web-app/pull/1514/files

Reply:
They just use a different external package. The one in here is the one we've had all along, but we were only really using it for viewing applicationData in the Template Builder. The newer one is one I made myself due to frustrations at the limitations of this one. The main one being that my new one allows you to edit whole JSON object at once if you want, which means we can copy-paste whole data view configs in one go, rather than adding them one field at a time (which you have to do here).